src/sentry/static/sentry/app/components/group/issuePluginActions.jsx

```diff
@@ -77,106 +98,88 @@ const IssuePlugin = React.createClass({
       state.error = {message: t('An unknown error occurred.')};
     }
     this.setState(state);
-  },
+  }
 
   fetchData() {
-    this.setState({
-      loading: true
+    this.onLoad(() => {
```

Comment:
added this to the constructor of the base class:
```javascript
if (this.fetchData) {
  this.fetchData = this.onLoad.bind(this, this.fetchData.bind(this));
}
if (this.onSubmit) {
  this.onSubmit = this.onSave.bind(this, this.onSubmit.bind(this));
}
```
so a component that inherits from it and uses `fetchData` and `onSubmit` as method names will just get that state management automatically